jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenRawConfigurationTest.java

```diff
@@ -101,7 +102,8 @@ public void testGetters() {
     Assert.assertEquals(
         Arrays.asList("--log", "info"), rawConfiguration.getProgramArguments().get());
     Assert.assertEquals(
-        new HashSet<>(Arrays.asList("additional", "tags")), rawConfiguration.getToTags());
+        new HashSet<>(Arrays.asList("additional", "tags")),
+        Sets.newHashSet(rawConfiguration.getToTags()));
```

Comment:
What's this change for?

Reply:
It's from error prone: "java.lang.Iterable does not have well-defined equals behavior"

Reply:
`rawConfiguration.getToTags()` returns iterable.

Reply:
I don't remember why `getToTags()` is returning `Iterable<String>` instead of `Set<String>`. Maybe we can try making it return `Set<String>`.